### ISSUE TITLE
Accessibility] [Keyboard Navigation -Playground]: Focus order is not logical after selecting the tabs in playground page. [Sev3][WCAG2.4.3]

### ISSUE THREAD

**Issue by Laxmidurga** [microsoft/tsdoc.org-website#36]
**Test Environment:**
OS Build: Windows 11
Version: 21H2 (OS Build 22000.318)
Browser: Edge
Version 96.0.1054.43 (Official build) (64-bit)
URL: https://tsdoc.org/
Screen reader: Narrator

**Repro Steps:**
1.	Open the above URL and login with valid credentials.
2.	TSDoc page will get displayed.
3.	Now, navigate to the ‘Playground’ link and activate it.
4.	Now, navigate to tabs (HTML, DOM, Line...) and activate it, check whether the focus order is logical or not after activating the tab.

**Actual:**
Focus order is not logical after selecting the tabs in playground page. After activating the tab, the focus is again moving to ‘TSDocs’ link in the header which is not logical.

**Expected:**
Focus order should be logical after selecting the tabs in playground page. After activating the tab, the focus should moves to the first interactive element in the tab.

**User Impact:**
When focus order does not follow the visible reading order, the experience can create a different interaction pattern, and/or unintended content relationships for assistive technology users. On rare occasions, there is a logical reason the focus order is different from the reading order, but the meaning of the content and interactions must remain the same.

**WCAG Reference:**
https://www.w3.org/WAI/WCAG21/Understanding/focus-order

https://user-images.githubusercontent.com/88436727/150644847-24dce685-acc2-4162-862a-c30e0b8f9922.mp4

**Comment by octogonz** [comment]
This issue has been closed because this Jekyll project has been retired. The documentation content has been migrated to Docusaurus. 

If the problem has not been addressed, please file a new issue under this GitHub project instead:  

https://github.com/microsoft/rushstack-websites
